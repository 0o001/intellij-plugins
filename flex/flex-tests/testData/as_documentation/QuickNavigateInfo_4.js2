package xxx {
  public class AAA {
    public static function set yyy(p:Object):void {}
  }
}

xxx.AAA.y<caret>yy = 1