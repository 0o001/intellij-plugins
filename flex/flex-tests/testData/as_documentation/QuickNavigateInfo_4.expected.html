(property) public static function xxx.AAA.yyy:Object